### PR TITLE
Switch from HTTP to gRPC server

### PR DESCRIPTION
**Pull request by kfyharukz**
Please check carefully only about:
- `cmd/moco-agent/cmd/server.go`
- `Dockerfile`
- `.github/workflow/release.yaml`

Other changes are:
1. Remove http API files
2. Rename gRPC API files cutting `_grpc` suffix

### CODE REVIEW

**Comment by kfyharukz** [comment]
note: the actual gRPC access log output
```
{"level":"info","ts":1614562946.836152,"logger":"grpc","msg":"finished unary call with code OK","peer.address":"[::1]:41528","grpc.start_time":"2021-03-01T01:42:26Z","grpc.request.deadline":"2021-03-01T01:42:27Z","system":"grpc","span.kind":"server","grpc.service":"grpc.health.v1.Health","grpc.method":"Check","peer.address":"[::1]:41528","grpc.code":"OK","grpc.time_ms":0.8220000267028809}
```

File: go.mod [hunk 1]
```diff
@@ -21,4 +22,5 @@ require (
 	golang.org/x/sync v0.0.0-20201207232520-09787c993a3a
 	google.golang.org/grpc v1.35.0
 	google.golang.org/protobuf v1.25.0
+	sigs.k8s.io/controller-runtime v0.8.1
```
Comment (masa213f) [inline go.mod:25]: Do you know why this module is listed?

Comment (kfyharukz) [reply go.mod:25]: It because my wrong import...
I'll replace it with `go.uber.org/zap`

**Review by masa213f — APPROVED** [review]
LGTM

**Pull request merged**